backend/src/routes/admin/option_tables/project_billing_resource_type_option.js

```diff
@@ -0,0 +1,40 @@
+const controller = require("@controllers/admin/option_tables");
+const validators = require("@validators/admin/option_tables");
+const what = "project_budget_resource_type_option";
+
+const routes = [
+  {
+    method: "GET",
+    url: `/${what}`,
+    schema: validators.getAll,
+    handler: controller.getAll,
+  },
+  {
+    method: "GET",
+    url: `/${what}/:id`,
```

Comment:
is the only difference between these option_tables routes the value of 'what' on Line 3?

probably. not worth it now, but that seems like it could be used to reduce duplicated code on these routes

Reply:
Yes it is.  The reason for this is in the future will need more lookup reference tables and they will need all these routes.